### ISSUE TITLE
Introduce Executor View

### ISSUE THREAD

**Issue by jyoungyun** [Samsung/ONE-vscode#1596]
## What

I would like to propose a new concept to support product file execution.
Product file generated through compilation of cfg files with toolchain can execute multiple functions such as inference or profile.
Now, product files can be inferred and profiled through ONE Explorer. However, considering the target device, it is not easy to express it as a toolchain and a target view. This is because the product file can be executed on the simulator of toolchain and the target device.

So I suggest the executor view.
It includes a simulator of toolchain and a target device.
Like tje toolchain view, the user can select the default executor. The default executor is then used to run the product file.
When a user installs a new toolchain, a simulator is automatically added. It is natural scenario to add toolchain and simulator when installing a toolchain. (If not, I am also considering changing the name of Toolchain view as Compiler(?).)

### Toolchain view

- For compilation
- Item : Toolchains
- Target file : config file (.cfg)
- Support default toolchain
   - Basically, the default toolchain compiles the selected config file.

### Executor view

- For execution
- Item : Simulator And Device target
- Target file : product file
- Support default executor
   - Basically, the default executor executes the product model file.

/cc @Samsung/one-vscode

**Comment by dayo09** [comment]
Could you give a detailed example for each views? How do you expect the trees look like?

My guess: Showing all the target files(==product files, *.tvn) in Executor view?

**Comment by jyoungyun** [comment]
> My guess: Showing all the target files(==product files, *.tvn) in Executor view?

Executor view contains the Simulator and device target.
And all the target files (product files) in ONE Explorer can be inferred and profiled using the executor in Executor views.

I want to attach my image file, but I can't due to in-house policy. I will update the image as soon as possible.

**Comment by dayo09** [comment]
@jyoungyun Thanks for the explanation. It's off topic but how did you upload the image? I've been failing uploading it and I also failed to get URL exception approval from IT center.
